sdk/schemaregistry/azure-schemaregistry-avroencoder/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Release History
 
+## 1.0.0b2 (Unreleased)
+
+### Features Added
+
+- `request_kwargs` has been added to `encode` and `decode` on `AvroEncoder` as an optional parameter to be passed into client requests.
+- The size of the current schema/schema ID caches will be logged at an info level when a new entry has been added.
```

Comment:
I can foresee the value of this logging info, but just wondering is this a feature request coming from customers or feature crew.

Reply:
I'm not sure but I can ask in the sync tomorrow.